features/livepatch.feature

```diff
@@ -73,14 +74,46 @@ Feature: Livepatch
         Supported livepatch kernels are listed here: https://ubuntu.com/security/livepatch/docs/kernels
         """
         Examples: ubuntu release
-            | release |
-            | focal   |
+            | release | machine_type |
+            | focal   | lxd-vm       |
+
+    @series.focal
+    @uses.config.machine_type.any
+    @uses.config.machine_type.gcp.generic
+    Scenario Outline: Attached livepatch status shows upgrade required when on an old kernel
+        Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token_staging` with sudo
```

Comment:
Do we need the staging token ?

Reply:
Yes, for now, because the production livepatch server doesn't support granular statuses yet